src/main/java/uk/gov/justice/digital/hmpps/keyworker/dto/PrisonKeyworkerConfiguration.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.keyworker.dto
+
+data class PrisonKeyworkerConfiguration(
+  val isEnabled: Boolean,
+  val hasPrisonersWithHighComplexityNeeds: Boolean,
+  val allowAutoAllocate: Boolean,
+  val capacityTier1: Int,
+  val capacityTier2: Int,
+  val kwSessionFrequencyInWeeks: Int,
+) {
+  companion object {
+    val NOT_CONFIGURED = PrisonKeyworkerConfiguration(false, false, false, 6, 9, 1)
```

Comment:
Just noticed that I think the capacityTier1 default used in the migration script in the previous PR was 1 rather than 6. Probably doesn't matter much as it's dev only but it's presumably detectable by them not having been migrated so could be changed to 6 with another migration